doc/cookbook/namedRoutes/NamedRoutes.lhs

```diff
@@ -0,0 +1,383 @@
+# Record-based APIs: the nested records case
+
+*Available in Servant 0.19 or higher*
+
+Servant offers a very natural way of constructing APIs with nested records, called `NamedRoutes`.
```

Comment:
Do you think it'd be good to clarify what "nested" means here? Describing the general pattern dividing up APIs in sub-APIs and being able to combine those easily, etc.

Reply:
Generic cookbook should have clarified this beforehand.